include/geojson/FeatureBuilder.hpp

```diff
@@ -277,7 +274,7 @@ namespace geojson {
                 }
             }
             else if (child.first == "id") {
-                id = child.second.data();
+                id = std::move(child.second.data());
```

Comment:
I _think_ that `std::move` might end up ripping stuff off off `child` here. It's hard to tell here without having example docs in front of me, but it looks like there's a chance that these `std::move` operations might modify the tree and bubble up the updates, harming reuse. There _might_ need to be some comments highlighting this so it's easier to identify while skimming.

Reply:
Once processed and recorded does the state of that node in the ptree really matter? The reference in the tree should still be valid if I understand the semantics of move correctly, but even if it isn't does it matter in this context?